compiler/desugared/linting.ml

```diff
@@ -35,8 +35,8 @@ let detect_empty_definitions (p : program) : unit =
           then
             Message.emit_spanned_warning
               (ScopeDef.get_position scope_def_key)
-              "In scope @{<yellow>\"%a\"@}, the variable @{<yellow>\"%a\"@} is \
-               declared but never defined; did you forget something?"
+              "In scope \"%a\", the variable \"%a\" is declared but never \
```

Comment:
I love it

Reply:
Maybe I put *too many* colors in there but now it's very easy to tweak them in `shared_ast/definitions.ml` ; feel free to tweak it if you find readability is impaired.